include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -89,6 +89,13 @@ namespace internal
     static const EvaluatorVariant variant = evaluate_evenodd;
   };
 
+  template <bool is_long>
+  struct EvaluatorSelector<MatrixFreeFunctions::tensor_none, is_long>
+  {
+    static const EvaluatorVariant variant =
+      evaluate_evenodd; // make it compile^^
```

Comment:
Since it's definitely not even-odd in that context, I suggest to use `evaluate_general` to not imply any particular structure.